### DISCUSSION TITLE
Personal Portfolio project hosted on GitHub pages having issue with link to another page (adds unnecessary .txt to the extension, resulting in a non existent page) help please!

### DISCUSSION THREAD

**Discussion by aijaza1** [community/community#57132] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [X] Projects

### Body

I have finished and tested everything about this website, I used React and NextJs to create this app. I tested on local host and "yarn dev" and everything works perfectly. Once I uploaded the code and deployed it using GitHub Pages, everything looks perfect except the button under my projects section. When you press more info for any of the projects it is adding a .txt extension at the end of the URL instead of correctly resolving to the corresponding JSX or JavaScript file.

Please take a look at my repo (https://github.com/aijaza1/aijaza1.github.io) all the code is there, the problem is in the "projects.jsx" file under "components" it takes the URL for the different pages and should take you to the correct path into the right file under the "pages" folder.

This is the last step and I have been struggling and have used lots of google and Chat GTP, please help!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @aijaza1 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
